packages/flutter/lib/src/rendering/object.dart

```diff
@@ -453,6 +453,30 @@ class PaintingContext extends ClipContext {
     return PaintingContext(childLayer, bounds);
   }
 
+  void _debugCheckForClipBehavior(String type) {
```

Comment:
this might be better named `_debugCheckForNoneClipBehavior` or some such, i was a bit confused at first as to why it was assuming the clip was "none" until i looked at all the call sites